css/evaluations.css

```diff
@@ -0,0 +1,110 @@
+body{
+	background-color: white;
+}
+
+.logoClass{
+	background-image: url('https://www.softvision.ro/images/common/logo-v2.png');
+	width: 200px;
+	height: 37.40px;
+	background-size: contain;
+	margin-left: 50px;
```

Comment:
One thing I like to do is to keep my related rules nicely bundled up. For example, `width` / `height` / `margin-*` are all layout-invalidating properties, so I'd keep them all together. Text formatting, colors, fonts etc I'd have in another area, possibly delimited by an empty line from other sets of rules. That way, it's easier to read through, since all of your related properties are in the same general area. Just a suggestion, feel free to organize your work however it makes the most sense for you.